### PR TITLE
gitignore .exe files

### PR DESCRIPTION
**Pull request by ukane-philemon**
This diff adds *.exe to list of gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
-vspd
+*.exe
```
Comment (jholdstock) [inline .gitignore:1]: No issue with adding `*.exe`, but why remove `vspd`?

Comment (ukane-philemon) [reply .gitignore:1]: vspd don't generate folder `vspd`? I assumed it was intended to ignore `vspd.exe` file.

Comment (jholdstock) [reply .gitignore:1]: On Linux the executable is not named `vspd.exe`, is named just `vspd` without a file extension.

Comment (ukane-philemon) [reply .gitignore:1]: Thanks for your feedback, very helpful.

**Review by dajohi — APPROVED** [review]

**Pull request merged**